### DISCUSSION TITLE
请问安卓版和桌面版中下载到本地的漫画分别保存在哪里？更新安卓版和桌面版要如何保证已下载的漫画不消失？

### DISCUSSION THREAD

**Discussion by 903208936** [xiaoyaocz/flutter_dmzj#209] [Q&A]
如题

**Comment by Huang2326** [comment]
桌面版（电脑版）的下载位置在C:\Users\Administrator\AppData\Roaming\com.xycz\flutter_dmzj\comic里面
安卓版的下载位置应该是在Android的data文件夹里面的对应包名文件夹里面
